fuel-vm/src/lib.rs

```diff
@@ -26,6 +33,7 @@ pub mod util;
 pub mod profiler;
 
 #[cfg(test)]
+#[cfg(feature = "alloc")]
```

Comment:
We can enable `alloc` in `dev-dependencies` and use it by default. Running tests without this feature doesn't make a lot of sense.

Reply:
Done in 90d00506